### PR TITLE
Fixed Dynmon example and logging strings

### PR DESCRIPTION
**Pull request by s41m0n**
Json template updated for Dynmon examples + minor spelling fixes

Signed-off-by: Simone Magnani <simonemagnani.96@gmail.com>

### CODE REVIEW

File: src/services/pcn-dynmon/src/swap/CodeRewriter.cpp [hunk 1]
```diff
@@ -289,13 +289,13 @@ void CodeRewriter::compile(std::string &original_code, ProgramType type,
   /*If is enabled, try enhanced compilation or basic compilation if it has failed.*/
   if(!is_enabled) {
     config = {};
-    logger->info("[Dynmon_CodeRewriter] No map marked as swappable, no compilation performed");
+    logger->info("[Dynmon_CodeRewriter] No map marked as swappable, no rewrites performed");
   } else if(try_enhance_compilation(original_code, type, maps_to_swap, config)) {
-    logger->info("[Dynmon_CodeRewriter] Successfully compiled with ENHANCED tuning");
+    logger->info("[Dynmon_CodeRewriter] Successfully rewrited with ENHANCED tuning");
```
Comment (frisso) [inline src/services/pcn-dynmon/src/swap/CodeRewriter.cpp:294]:
```suggestion
    logger->info("[Dynmon_CodeRewriter] Successfully rewritten with ENHANCED tuning");
```

File: src/services/pcn-dynmon/src/swap/CodeRewriter.cpp [hunk 2]
```diff
@@ -289,13 +289,13 @@ void CodeRewriter::compile(std::string &original_code, ProgramType type,
...
   } else if(do_base_compile(original_code, maps_to_swap, config)) {
-    logger->info("[Dynmon_CodeRewriter] Successfully compiled with BASE tuning");
+    logger->info("[Dynmon_CodeRewriter] Successfully rewrited with BASE tuning");
```
Comment (frisso) [inline src/services/pcn-dynmon/src/swap/CodeRewriter.cpp:296]:
```suggestion
    logger->info("[Dynmon_CodeRewriter] Successfully rewritten with BASE tuning");
```

**Review by frisso — APPROVED** [review]

**Review by palexster — APPROVED** [review]

File: src/services/pcn-dynmon/src/swap/CodeRewriter.cpp (hunk 1 above)
Comment (s41m0n) [reply src/services/pcn-dynmon/src/swap/CodeRewriter.cpp]: Done, thanks :)

File: src/services/pcn-dynmon/src/swap/CodeRewriter.cpp (hunk 2 above)
Comment (s41m0n) [reply src/services/pcn-dynmon/src/swap/CodeRewriter.cpp]: Done, thanks :)

**Pull request merged**